parts/agentparams.t

```diff
@@ -24,6 +24,34 @@
       }, 
       "type": "string"
     },
+    "{{.Name}}osImageOffer": {
+      "defaultValue": "UbuntuServer",
+      "metadata": {
+        "description": "Linux OS image type."
+      },
+      "type": "string"
+    }, 
+    "{{.Name}}osImagePublisher": {
+      "defaultValue": "Canonical",
+      "metadata": {
+        "description": "OS image publisher."
+      },
+      "type": "string"
+    }, 
+    "{{.Name}}osImageSKU": {
+      "defaultValue": "16.04-DAILY-LTS",
```

Comment:
This image was deleted last week so we had to replace it with the monthly version (16.04-LTS). Please see #1755.

Reply:
Updated.